app/assets/stylesheets/catarse_bootstrap/_extra.sass

```diff
@@ -187,3 +187,15 @@ body
   -webkit-filter: blur(15px) brightness(0.75)
   background-position: 50% 50%
   background-size: cover
+
+.discover-form
```

Comment:
Let's try to use a compass mixin here: http://compass-style.org/examples/compass/css3/input-placeholder/